ansible/roles/connectbox-pi/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set keyboard to the US layout
+  lineinfile:
+    path: /etc/default/keyboard
+    regexp: '^XKBLAYOUT="gb"'
+    line: 'XKBLAYOUT="us"'
+
```

Comment:
This doesn’t apply the change, and we want to avoid forcing a reboot, so you should add a handler here to restart the `keyboard-setup` service.

Handlers are a common ansible thing, so it’s a good intro :-)

Docs: https://docs.ansible.com/ansible/playbooks_intro.html#handlers-running-operations-on-change . There’s also an existing example in the connectbox-pi role for restarting `php5-fpm` as a reference point.